Backend/TimeTracker.Api/Controllers/TimerController.cs

```diff
@@ -0,0 +1,206 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using TimeTracker.Api.Database;
+using TimeTracker.Api.Database.Models;
+using TimeTracker.Api.DTOs;
+using TimeTracker.Api.Helpers;
+
+namespace Backend.Controllers
+{
+    [ApiController]
+    [Authorize]
+    [Route("[controller]")]
+    public class TimerController : ControllerBase {
+        MainDb database;
+        AuthHelper authHelper;
+
+        public TimerController(MainDb db, AuthHelper auth)
+        {
+            database = db;
+            authHelper = auth;
+        }
+
+        // returns a TimerDTO for the newly created Timer
+        [HttpPost]
+        [Route("Start")]
+        public async Task<GenericResponseDTO<TimerDTO>> StartTimer(TimerCreateDTO timerInfo) {
+
+            if(User == null) {
+                return new GenericResponseDTO<TimerDTO> {
+                    Message = "Not logged in.",
+                    Success = false
+                };
+            }
+
+            Project project = await database.Projects
+                .FirstOrDefaultAsync(p => p.Id == timerInfo.ProjectId);
+
+            if(project == null) {
+                return new GenericResponseDTO<TimerDTO> {
+                    Message = "Could not find project.",
+                    Success = false
+                };
+            }
+
+            Timer timer = (await database.Timers
+                .AddAsync(new Timer {
+                    User = await authHelper.GetCurrentUser(User, database),
+                    StartTime = DateTime.Now,
```

Comment:
`DateTime.Now` is time zone dependent, so if our server was for some reason in California this time would be completely different than what you would expect. It's best practice to just save all dates in UTC format. I think its `DateTime.UTCNow`